packages/styles/notice.css

```diff
@@ -8,7 +8,9 @@
   --notice-background-color: var(--notice-info-color);
   --notice-border-color: var(--accent-dark);
   --notice-link-hover-color: var(--accent-medium);
-  --notice-icon-size: 1.2em;
+  --notice-icon-size: 1rem;
+  --notice-border-thickness: 1px;
+  --notice-border-radius: 4px;
```

Comment:
I don't think we want these to be configurable, so these can just be set via properties of the component.

```suggestion
```